debian/postinst

```diff
@@ -67,8 +68,31 @@ upgrade_to_status_cache() {
     ua status 2>&1 > /dev/null
 }
 
+is_release_upgrade() {
+    # Return 0 if the package version we upgrade from does not match the
+    # current series (e.g. 14.04)
+    if test "${2#*$VERSION_ID}" = ${2}; then
```

Comment:
`${2#*$VERSION_ID}` is a pretty convoluted way of performing this check, I think; I had to double-check several times that it was doing the right thing.  Perhaps something like this would be clearer:

```sh
pkg_version=$2  # this is assuming that $2 is correct, for now
case $pkg_version in
    *${VERSION_ID}*) return 1;;
esac
return 0
```

IMO, the pattern matching makes it clearer what we're actually trying to do here (which is a substring check, basically).

Reply:
good point. done.